nagios/conf.d/misc.cfg

```diff
@@ -109,3 +109,14 @@ define service {
 	service_description kibana-web
 	check_command check_https_sticky!-e 401
 }
+
+
+define service {
+	host_name		perf-bootstrap
+	service_description     influxdb-database-backup
+	active_checks_enabled   0
+	passive_checks_enabled  1
+	check_freshness         1
+	freshness_threshold     93600 ; 26 hour threshold, since backups may not always finish at the same time
+	check_command           no-perf-harness-influxdb-backup-report
```

Comment:
You don't have to define new command for dummy checks. You can just use `return-ok`. See `errors-in-log` service.